packages/shared/src/utils/constants/jurisdictions-config.ts

```diff
@@ -11,3 +13,63 @@ export const Platforms = Object.freeze({
     MT5: 'mt5',
     DERIVEZ: 'derivez',
 });
+
+export const getFormattedJurisdictionCode = (jurisdiction_code: string) => {
+    let formatted_label = '';
+
+    switch (jurisdiction_code) {
+        case Jurisdiction.SVG:
+            formatted_label = localize('SVG');
+            break;
+        case Jurisdiction.BVI:
+            formatted_label = localize('BVI');
+            break;
+        case Jurisdiction.LABUAN:
+            formatted_label = localize('Labuan');
+            break;
+        case Jurisdiction.VANUATU:
+            formatted_label = localize('Vanuatu');
+            break;
+        case Jurisdiction.MALTA_INVEST:
+            formatted_label = localize('Malta');
+            break;
```

Comment:
👀 I don't think Landing Companies needs to be localize? Can you check with the translation team?

Reply:
In production, Landing company names are localized